packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -498,18 +489,13 @@ export default defineComponent({
       }
     }
 
-    // Focus and keyboard navigation
-
-    const isFocusedComputed = computed(() => {
-      // If we show dropdown content that means select is focused
-      return isFocused.value || showDropdownContent.value
-    })
+    const openDropdown = () => {
+      if (props.disabled || props.readonly) { return }
+      showDropdown()
+    }
 
     const onSelectClick = () => {
-      if (props.disabled) {
-        return
-      }
-
+      if (props.disabled || props.readonly) { return }
```

Comment:
Let's choose one of this readonly behaviors: 

Hide arrow icon. 😐
![image](https://user-images.githubusercontent.com/23530004/152867107-3e4d35dd-1556-432c-b976-ca51ca2d6e6e.png)

Show select options, but don't allow user to click on any options. 👍
![image](https://user-images.githubusercontent.com/23530004/152867335-39b8591d-54a0-4b8e-a506-33fbd4ac1565.png)

Reply:
After discussion, we accepted the first option (with the icon color changed to inactive)